src/me/corriekay/pokegoutil/BlossomsPoGoManager.java

```diff
@@ -1,21 +1,68 @@
 package me.corriekay.pokegoutil;
 
+import java.io.IOException;
+
+import javax.swing.SwingUtilities;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.image.Image;
+import javafx.stage.Stage;
+import javafx.stage.StageStyle;
 import me.corriekay.pokegoutil.controllers.AccountController;
+import me.corriekay.pokegoutil.utils.Config;
 import me.corriekay.pokegoutil.utils.Console;
 import me.corriekay.pokegoutil.utils.Utilities;
 
 
-public class BlossomsPoGoManager {
+public class BlossomsPoGoManager extends Application{
+	
+	public static final String VERSION = "0.2.0-Alpha";
```

Comment:
Can you please change it back to the current alpha? \*lol\*
Or better - that saves us one PR, change it to `v0.1.2-alpha.2`, which should be the next release. At least like planned now.